Upstream: yes
Source: http://bugs.python.org/issue13007

hunk for Misc/NEWS is removed

# HG changeset patch
# User Jesus Cea <jcea@jcea.es>
# Date 1316444238 -7200
# Node ID 14cafb8d148037724bbb55f4f65e45812bf5a44c
# Parent  9f3595fb6c92b9dbe3cbbfdf32e431bab206ee0a
Close #13007: whichdb should recognize gdbm 1.9 magic numbers

diff --git a/Lib/whichdb.py b/Lib/whichdb.py
--- a/Lib/whichdb.py
+++ b/Lib/whichdb.py
@@ -91,7 +91,7 @@ def whichdb(filename):
         return ""
 
     # Check for GNU dbm
-    if magic == 0x13579ace:
+    if magic in (0x13579ace, 0x13579acd, 0x13579acf):
         return "gdbm"
 
     # Check for old Berkeley db hash file format v2
